packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -35,29 +36,39 @@ const SquiggleViewerBody: FC<{ value: SqValue }> = ({ value }) => {
     return <MessageAlert heading="Focused variable is not defined" />;
   }
 
-  return <ExpressionViewer value={valueToRender} />;
+  return <ExpressionViewer value={valueToRender} isFocused={!!focused} />;
```

Comment:
You can add:
```
function useIsFocused() {
  const { focused } = useViewerContext();
  return !!focused;
}
```

To `ViewerProvider.tsx`.

And then use it in `VariableBox`. Then you won't have to pass it through `ExpressionViewer`.

Reply:
Cool, added. However, we need to check it for each VariableBox. I did it as,
```js
export function useIsFocused(location: SqValueLocation) {
  const { focused } = useViewerContext();
  return !!focused && locationAsString(focused) === locationAsString(location);
}
```